hello_world.py

```diff
@@ -0,0 +1,5 @@
+import sys
```

Comment:
It is a good practice to display the python version used when running a script

Reply:
Thanks for the comment, Erick!